# install to /usr/games
Index: dwarf-therapist-21.8/dwarftherapist.pro
===================================================================
--- dwarf-therapist-21.8.orig/dwarftherapist.pro	2014-06-20 15:50:53.000000000 +0200
+++ dwarf-therapist-21.8/dwarftherapist.pro	2014-06-22 04:22:23.348736034 +0200
@@ -86,10 +86,10 @@
     SOURCES += src/dfinstancelinux.cpp
 
     message(Setting up for Linux Install)
-    target.path = /usr/bin
+    target.path = /usr/games
     INSTALLS += target
 
-    bin.path = /usr/bin
+    bin.path = /usr/games
     bin.files += bin/release/DwarfTherapist
     bin.files += dist/dwarftherapist
     INSTALLS += bin
@@ -98,8 +98,8 @@
     application.files = dist/dwarftherapist.desktop
     INSTALLS += application
 
-    doc.path = /usr/share/doc/dwarftherapist
-    doc.extra = cp LICENSE $(INSTALL_ROOT)/usr/share/doc/dwarftherapist/copyright; cp README.md $(INSTALL_ROOT)/usr/share/doc/dwarftherapist/README.Debian
+    doc.path = /usr/share/doc/dwarf-therapist
+    ;doc.extra = cp LICENSE $(INSTALL_ROOT)/usr/share/doc/dwarftherapist/copyright; cp README.md $(INSTALL_ROOT)/usr/share/doc/dwarftherapist/README.Debian
     INSTALLS += doc
 
     icon.path = /usr/share/pixmaps
@@ -107,7 +107,7 @@
     icon.files += img/dwarftherapist.xpm
     INSTALLS += icon
 
-    share.path = /usr/share/dwarftherapist
+    share.path = /usr/share/games/dwarf-therapist
     share.extra = mkdir -p $(INSTALL_ROOT)/usr/share/dwarftherapist; \
         mkdir -p $(INSTALL_ROOT)/usr/share/dwarftherapist/memory_layouts; \
         mkdir -p $(INSTALL_ROOT)/usr/share/dwarftherapist/memory_layouts/linux; \
Index: dwarf-therapist-21.8/dist/dwarftherapist
===================================================================
--- dwarf-therapist-21.8.orig/dist/dwarftherapist	2014-06-20 15:50:53.000000000 +0200
+++ dwarf-therapist-21.8/dist/dwarftherapist	2014-06-22 04:08:39.484760541 +0200
@@ -1,5 +1,5 @@
 #!/bin/bash
-cd /usr/share/dwarftherapist
+cd /usr/share/games/dwarf-therapist

 #Search for the appropriate SUDO application
 _foundprog=
@@ -19,9 +19,9 @@

 if [ -f /proc/sys/kernel/yama/ptrace_scope ] && [ `cat /proc/sys/kernel/yama/ptrace_scope` -eq 1 ]; then
 	echo Dwarf therapist needs to run as root because user ptrace is disabled
-	$SUDO /usr/bin/DwarfTherapist
+	$SUDO /usr/games/DwarfTherapist
 else
-	/usr/bin/DwarfTherapist
+	/usr/games/DwarfTherapist
 fi


Index: dwarf-therapist-21.8/dist/dwarftherapist.desktop
===================================================================
--- dwarf-therapist-21.8.orig/dist/dwarftherapist.desktop	2014-06-20 15:50:53.000000000 +0200
+++ dwarf-therapist-21.8/dist/dwarftherapist.desktop	2014-06-22 04:48:47.800688900 +0200
@@ -1,12 +1,9 @@
 [Desktop Entry]
-Encoding=UTF-8
 Name=Dwarf Therapist
-Version=0.6.5
 Comment=Dwarf Viewer and Manager for Dwarf Fortress
 Exec=dwarftherapist
 TryExec=dwarftherapist
 Icon=dwarftherapist
 Terminal=false
 Type=Application
-Categories=Qt;GNOME;Utility;
-
+Categories=Game;
